crypto/ecdsa.c

```diff
@@ -1312,3 +1354,25 @@ int ecdsa_recover_pub_from_sig(const ecdsa_curve *curve, uint8_t *pub_key,
 #endif
   return tc_ecdsa_recover_pub_from_sig(curve, pub_key, sig, digest, recid);
 }
+
+int ecdh_multiply(const ecdsa_curve *curve, const uint8_t *priv_key,
+                  const uint8_t *pub_key, uint8_t *session_key) {
+#ifdef USE_SECP256K1_ZKP_ECDSA
+  if (curve == &secp256k1) {
+    return zkp_ecdh_multiply(curve, priv_key, pub_key, session_key);
+  }
+#endif
+  return tc_ecdh_multiply(curve, priv_key, pub_key, session_key);
+}
+
+ecdsa_tweak_pubkey_result ecdsa_tweak_pubkey(const ecdsa_curve *curve,
+                                             const uint8_t *pub_key,
+                                             const uint8_t *priv_tweak,
```

Comment:
```suggestion
                                             const uint8_t *tweak,
```

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4083/commits/f6675fd5ecf272faa011997889fc9ad768ebe0d7.